rest-api-spec/src/main/resources/rest-api-spec/api/msearch.json

```diff
@@ -5,6 +5,10 @@
       "description":"Allows to execute several search operations in one request."
     },
     "stability":"stable",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/x-ndjson"]
```

Comment:
Since this technically supports sending `application/json` too, should we add it after the preferred `application/x-ndjson`? I can see an argument against not including it as even though the `content-type` header is `application/json`, the request body format must be newline delimited JSON